packages/build-utils/test/fixtures/15-npm-8-legacy-peer-deps/package-lock.json

```diff
@@ -7,6 +7,9 @@
       "dependencies": {
         "react": "16.8.0",
         "swr": "1.3.0"
+      },
+      "engines": {
+        "node": "16.x"
```

Comment:
@styfle you were the last to touch this one. Not sure why this is getting generated but wasn't before. I've tried reverting, but it generates when I run the unit tests locally.

Reply:
Its probably fine if the tests pass.

The test is checking to see if `npm install` fails so I'm a little surprised if the lock file updates

See https://github.com/vercel/vercel/pull/8598